packages/ui/src/proposals/components/ProposalDiscussions.tsx

```diff
@@ -26,7 +26,7 @@ interface Props {
 }
 
 const hints = {
-  open: 'This is an unmoderated discussioon, everyone can comment.',
+  open: 'This is an open discussion, everyone can comment.',
```

Comment:
[according to](https://github.com/Joystream/pioneer/issues/3900#issuecomment-1406287531)
```suggestion
  open: 'Open discussion is open for every member to comment on, while closed would only be available for council members, white-listed members and creator of the proposal.',
```